src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -173,6 +173,26 @@ then you have to download the database driver and put it at: *PROACTIVE_HOME/add
 
 Scheduler, and 3 microservices (*scheduling-api*, *job-planner*, and *proactive-cloud-watch*) have to point to the same database.
 
+[[_backup]]
+=== Configure automated backup
+
+We provide an internal backup mechanism for the database,
+in case HSQLDB is used, and for data files (workflows, logs).
+In case another DB is used, it is possible to combine a vendor specific backup mechanism
+with the internal mechanism for data files.
+
+To use the automated backup, edit the file `PROACTIVE_HOME/config/shared/settings.ini`.
+Enable backup mechanism by setting `pa.server.backup` to true.
+Backup will be performed according to provided cron expression in `pa.server.backup.period`.
+Set file/folders you want to backup into `pa.server.backup.targets`.
+Backup will appear in the folder specified by `pa.server.backup.destination`.
+
+The backup mechanism automatically freezes the scheduler
```

Comment:
NOTE: The backup mechanism automatically freezes ...
It expects currently running tasks to terminate in a reasonable amount of time (configurable using the pa.server.backup.possile.delay property). In case a currently running task does not terminate after the expected delay, backup will not be performed.